utilities/bash/run_status

```diff
@@ -32,6 +32,31 @@ sixdeskmess -1 "Checking Study $LHCDescrip"
 # DON'T LOCK unless we are modifying.....
 # at the expense of possibly inconsistent reports
 
+
```

Comment:
Excellent bunch of lines! To improve them I would:
- call the sixdeskSetBoincVars function in dot_profile - it is useless that we have there a super-refined set-up of the environment if then we don't load it when needed and we use hard-coded values - eg for the server name...
- maybe the two loops could become only one ... but I don't know the format of the output, so maybe it is cleaner as you did :)